core/runtime/src/main/java/io/quarkus/runtime/configuration/ConfigInstantiator.java

```diff
@@ -28,19 +31,33 @@
 public class ConfigInstantiator {
 
     private static final Pattern COMMA_PATTERN = Pattern.compile(",");
+    // certain well-known classname suffixes that we support
+    private static Set<String> supportedClassNameSuffix;
+
+    static {
+        final Set<String> suffixes = new HashSet<>();
```

Comment:
Do you mind avoiding using final on local variables. I don't think we use that approach elsewhere and in your case it shows artificial line changes compared to the original code which makes it harder to read.

Reply:
@emmanuelbernard, I'll keep that in mind for unchanged lines going forward.